src/composables/approvals/useTokenApprovalActions.spec.ts

```diff
@@ -28,7 +28,7 @@ test('Shows correct labels', async () => {
   );
 });
 
-const spender = '0x8c9e6c40d3402480ACE624730524fACC5482798c';
+const spender = randomAddress();
```

Comment:
Spender will typically be the Vault address or a relayer address, so I'm not sure if setting it to a random address makes sense here. But I guess it also doesn't matter.